test/buffer/worker/001_send_records_test.go

```diff
@@ -26,24 +27,24 @@ func (ts *testSuite) test001SendPayload() {
 	}
 
 	// Check API metrics
-	var recordsMetric float64
-	var bodyBytesMetric float64
+	recordsMetric := atomic.NewFloat64(0)
+	bodyBytesMetric := atomic.NewFloat64(0)
```

Comment:
Tu mi test hlasi data-race, aj ked myslim, ze ziadna tu nieje.

Po jednom sa iteruje `ch` a potom sa skontroluje sucet, nedeje sa tu nic paralelne.

Ale mozno preto, ze je to v callbacku, tak to zle detekuje? :man_shrugging: